cmd/monaco/generate/deletefile/deletefile_test.go

```diff
@@ -279,7 +279,7 @@ func TestDoesNotOverwriteExistingFiles(t *testing.T) {
 
 	// AND THEN new DOT file is created with timestamp appended
 	time := timeutils.TimeAnchor().Format("20060102-150405")
-	newFile := fmt.Sprintf("delete.yaml_%s", time)
+	newFile := fmt.Sprintf("%s_delete.yaml", time)
```

Comment:
```suggestion
	newFile := fmt.Sprintf("delete_%s.yaml", time)
```
Wouldn't it be easier to log the timestamp after the delete? With that, autocompletion can already suggest what files are delete-files. 

We do it similarly with downloads (`download_{timestamp}`).

Reply:
As you can define your own filenames, it's not just "after the `delete`", it would mean splitting the string on the last dot and placing the timestamp between the actual name and extension.

Can be done, but not sure if it's that much better tbh. Prepended timestamps can also make sorting through the files a bit easier

Reply:
Addressed by inserting between filename and file-ending (before the last dot; if one exists) as discussed internally